### PR TITLE
New Projection and static utilities

### PR DESCRIPTION
**Pull request by bombaci-vsc**
Here are some changes which add a Contour to Image projection and refactors some existing projection functions to not require creating a projection model object

### CODE REVIEW

**Comment by asylvest** [comment]
Thanks.  Do you have an ETA for when the equations for the fullContourToImageCoord() will appear in an updated SICD Volume 3?  Is there a draft copy that you could send?

**Comment by chvink** [comment]
looks like we'll want to regenerate scene's python bindings, too?

File: modules/c++/scene/include/scene/ProjectionModel.h [hunk 1]
```diff
@@ -95,6 +95,18 @@ class ProjectionModel
      *  R/Rdot Contour Ground Plane Intersection
      *
      */
+    static Vector3
```
Comment (asylvest) [inline modules/c++/scene/include/scene/ProjectionModel.h]: I'm not necessarily opposed to having a static method like this, but what's the use case where you want to just get the contour but don't want to construct a ProjectionModel?

**Comment by asylvest** [comment]
@chvink let's wait until the dust settles on the C++ side but yeah once that's finalized we'll want to re-generate the Python bindings.

File: modules/c++/scene/include/scene/ProjectionModel.h (hunk 1 above)
Comment (mtrachy) [inline modules/c++/scene/include/scene/ProjectionModel.h]: The main use case I can think of is when doing projections prior to (or during) image formation, when the entirety of the information needed to construct a ProjectionModel is not available.

**Comment by asylvest** [comment]
OBE - if this ever is something we want to move into SIX, we'll certainly want to revisit the implementation that was started here (hopefully with the corresponding equations documented in the SICD spec)

**Pull request closed without merging**